app/workers/notifications/new_comment_worker.rb

```diff
@@ -0,0 +1,11 @@
+module Notifications
+  class NewCommentWorker
+    include Sidekiq::Worker
```

Comment:
[CodeClimate](https://codeclimate.com/github/thepracticaldev/dev.to/pull/5565) is saying that this and the other Notification workers have similar blocks of code and that we should refactor them. I think you are fine to ignore it.

Reply:
yeah, it's fine